region/maridia/outer/Fish Tank.json

```diff
@@ -349,6 +329,12 @@
       "link": [1, 5],
       "name": "Cross Room Jump to the Frozen Fish",
       "notable": false,
+      "entranceCondition": {
+        "comeInJumping": {
+          "speedBooster": true,
+          "minTiles": 3.4375
```

Comment:
This is an extra ~1 tile compared to what was in there before, but I couldn't get past the ledge with anything less. If it's possible, it may need to be notable. (same for the spring ball variant)
Most of the other cross room jumps from this door I was able to tighten a bit.

Reply:
@blkerby mentioned to me that it is helpful to be in a downward pose before hitting the turnaround. This isn't how I've been testing them, so there are probably some that should be changed (fortunately most of the runway lengths were based on bobbob's videos, so those should be correct).

Reply:
I went back through and it looks like this shouldnt change the number of tiles for any of the other strats that I migrated